### ISSUE TITLE
Problems when using modal

### ISSUE THREAD

**Issue by grzegorzgolec** [valor-software/ngx-bootstrap#775]
I have installed 1.0.24 from npm, and Iam trying to use Modal. Iam getting error all the time (as on picture below)

![modal](https://cloud.githubusercontent.com/assets/6419370/17080554/8226d9a6-5134-11e6-9ca1-56dd98bca9fa.png)

How to use it ?

**Comment by adrianfaciu** [comment]
Did you follow all the instructions from the [demo usage](http://valor-software.com/ng2-bootstrap/#/modals) ? If yes, can you provide a small sample on how to reproduce this ?

**Comment by valorkin** [comment]
please show component containing this html

**Comment by grzegorzgolec** [comment]
Yes i have followed tutorial. I have also checked directory structure of package installed from npm. The main difference I saw was that "ng2-bootstrap\components\modal\modal.component.d" have no Directive annotation as in tutorial
<code>
@Directive({
  selector: '[bsModal]',
  exportAs: 'bs-modal'
})
</code>

Error that i got is strictly says "There is no directive with exportAs". Is npm version is actual ?
